.github/workflows/pyoxidizer.yml

```diff
@@ -0,0 +1,178 @@
+name: Build PyOxidizer Binary
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build-linux-gnu:
+    name: Build Linux (GNU)
+    runs-on: ubuntu-latest
+    container: quay.io/pypa/manylinux2014_x86_64
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/pyoxidizer/
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+          key: linux-cache
+
```

Comment:
Why don't you do a 
```
     - uses: actions/setup-python@v4
     ...
```
here? Saves on all the explicit paths.

Reply:
Unfortunately `actions/setup-python` depends on a newer LIBC and is incompatible with this container. Same for the other job.

Reply:
Maybe this as a comment on why it is not used.